subsys/mgmt/mcumgr/smp_bt.c

```diff
@@ -89,62 +103,48 @@ static struct conn_param_data *get_conn_param_data(const struct bt_conn *conn)
 }
 
 /* Sets connection parameters for a given conn. */
-static void set_conn_latency(struct bt_conn *conn, bool low_latency)
+static void conn_params_set(struct bt_conn *conn, bool smp_params)
```

Comment:
Maybe change the `smp_params` to `const struct bt_le_conn_param` and pass the `SMP_BT_CONN_PARAMS` or `DEFAULT_CONN_PARAMS` directly? Or at least rename the `smp_params` to something like `set_default` and reverse logic?